src/lib/google-auth/GoogleAuth.svelte

```diff
@@ -73,22 +78,27 @@
   }
 
   function initialise () {
-    gapi.load('auth2', async () => {
-      googleAuth = gapi.auth2.init({ client_id: clientId })
-      googleAuth.then(attachHandler, handleInitialisationError)
+    google.accounts.id.initialize({
+      client_id: clientId,
+      scope,
+      callback: attachHandler
     })
-  }
 
-  function attachHandler () {
-    googleAuth.attachClickHandler(signinCta, {},
-      () => dispatch('auth-success', { user: googleAuth.currentUser.get() }),
-      e => dispatch('auth-failure', { error: e })
-    )
     disabled = false
   }
 
-  function handleInitialisationError (e) {
-    console.error('gauth initialisation error', e)
-    dispatch('init-error', { error: e })
+  function handleClick () {
+    google.accounts.id.prompt()
+  }
+
+  function attachHandler ({ credential }) {
+    if (!credential) {
+      dispatch('auth-failure', { error: 'Unable to get credentials' })
+      return
+    }
+
+    const { name, email, picture } = decode(credential)
+
+    dispatch('auth-success', { user: { name, email, picture } })
```

Comment:
How do we get their Google id to write to their profile?

Reply:
I believe we can use the `sub` (subject) property of the JWT token:
https://developers.google.com/identity/gsi/web/reference/js-reference#credential

Reply:
Added `sub` renamed as `id` on output of `auth-success` ✅